### PR TITLE
Remove AA attack limit of half dice side value

### PR DESCRIPTION
**Pull request by ron-murhammer**
Remove check that limits aa attack to half of dice sides as it isn't clear as to why that is done. Given that check exists, no existing maps should have a value greater than half so should have no negative impact.

Here is the original commit that added that code: https://github.com/triplea-game/triplea/commit/8a5f0e2c43a013d14f695050cfd340ecc70f98be#diff-b277d7a65530ad0e35446534517831f9

@veqryn Any background on this? Its a bit hard to follow why that AA attack cap is needed. It looked like there was originally some sort of "groupSize" used when rolling LL but that seems to not exist anymore: https://github.com/triplea-game/triplea/commit/8a5f0e2c43a013d14f695050cfd340ecc70f98be#diff-b277d7a65530ad0e35446534517831f9R126

### CODE REVIEW

**Comment by veqryn** [comment]
If memory serves correctly, it is because of the remainder issue.

"All attack values (with and without radar) should divide into diceSides without any remainder, otherwise the players will get errors during LowLuck."

If dice sides is 6, you can do 6/1, 6/2, and 6/3 with no remainder, but you can't do 6/4 because there would be a remainder.

I haven't kept up on things, so I'm not sure if this was solved or how.

**Comment by ron-murhammer** [comment]
@veqryn Yeah, I saw that in POS2 but not clear why a remainder is a problem or why you'd be dividing dice sides by the AA attack to potentially get the remainder.

**Comment by Cernelius** [comment]
If the matter is not having the remainder, then a check limiting to half of dice sides is not a solution, anyways, since that does not assure not having a remainder. I guess this was done back then when only dice sides = 6 were possible.

**Comment by ron-murhammer** [comment]
@RoiEXLab While I almost never comment out code, given this is kind of an experimental change, I'd like to keep it for now so I don't forget about it. I'll look to make a follow up PR to either remove or add a better check back in over the next few weeks.
